apps/backoffice-v2/src/pages/Entity/hooks/useEntity/utils.ts

```diff
@@ -77,10 +77,18 @@ export const extractCountryCodeFromWorkflow = (workflow: TWorkflowById) => {
   })?.issuer?.country;
 };
 
-export const omitPropsFromObject = (obj, ...props) => {
+export const omitPropsFromObject = <
```

Comment:
`omitPropsFromObject` now autocompletes to keys available on the passed object, and the type of the output object is now the object without the omitted props.